msbuild/tests/Xamarin.iOS.Tasks.Tests/FrameworkListTest.cs

```diff
@@ -29,12 +29,12 @@ public void CheckFrameworkListFile (string frameworkListFile)
 
 			foreach (var assembly in frameworkListAssemblies) {
 				if (!installedAssemblies.Any (a => a.Name == assembly.Name))
-					ReportAssemblies (assembly, $"Incorrectly added assemblies in '{frameworkListFile}'");
+					ReportAssemblies (assembly, $"Incorrectly listed assemblies in '{frameworkListFile}'. Check if those assemblies/facades exist in the final SDK root folder.");
 			}
 
 			foreach (var assembly in installedAssemblies) {
 				if (!frameworkListAssemblies.Any (a => a.Name == assembly.Name))
-					ReportAssemblies (assembly, $"Missing assemblies in '{frameworkListFile}'");
+					ReportAssemblies (assembly, $"Missing assemblies in '{frameworkListFile}'. We have detected assemblies/facades in the SDK root folder that are not listed in the 'FrameworkList' file.");
```

Comment:
This is already quite good, maybe just align it with the above message if you go with my suggestion:
```
One or more assemblies in the the SDK root folder are not listed in '{frameworkListFile}'.  Update the list if an assembly was intentionally added.
```